html/semantics/embedded-content/the-iframe-element/support/download_stash.py

```diff
@@ -4,6 +4,11 @@ def main(request, response):
     token = request.GET[b"token"]
     response.status = 200
     response.headers.append(b"Content-Type", b"text/html")
+
+    // Make sure to disable sniffing because it would read enough bytes to finish
+    // the load, before even giving the client application a chance to cancel it.
```

Comment:
```suggestion
    # Make sure to disable sniffing because it would read enough bytes to finish
    # the load, before even giving the client application a chance to cancel it.
```